config/karma/shared.karma.conf.js

```diff
@@ -14,7 +14,7 @@ function getConfig(config) {
   const path = require('path');
   let testWebpackConfig = require('../webpack/test.webpack.config');
   let remapIstanbul = require('remap-istanbul');
-  let skyPagesConfig = require('../sky-pages/sky-pages.config').getSkyPagesConfig(argv.command);
+  let skyPagesConfig = require('../sky-pages/sky-pages.config').getSkyPagesConfig(argv._[0]);
```

Comment:
Non-blocking comment regarding readability: I wasn't aware of the underscore property being attached to `argv` (by minimist?) and had to look it up. It'd probably make more sense to assign the value to a human-readable variable, and pass _that_ into the method. Something like:

```
// Some comment describing what `argv._` is...
const cliCommand = argv._[0];
const skyPagesConfig = require('../sky-pages/sky-pages.config').getSkyPagesConfig(cliCommand);
```